web/frontend/src/components/navigation/components/nav.svelte

```diff
@@ -23,54 +27,165 @@ const handleRemoveWaypoint = async (id: string) => {
   }
 };
 
+const handleAddObstacle = () => {
+  $obstacles = [
+    createObstacle(`Obstacle ${$obstacles.length + 1}`, $mapCenter.lng, $mapCenter.lat),
+    ...$obstacles,
+  ];
+};
+
+const handleLngLatInput = (index: number, event: CustomEvent<LngLat>) => {
+  $obstacles[index]!.location = event.detail;
+};
+
+const handleDeleteObstacle = (index: number) => {
+  $obstacles = $obstacles.filter((_, i) => i !== index);
+};
+
+const handleTabSelect = (event: CustomEvent) => {
+  $tab = event.detail.value;
+};
+
+const handleGeometryInput = (index: number, geoIndex: number) => {
+  return (event: CustomEvent<Geometry>) => {
+    $obstacles[index]!.geometries[geoIndex] = event.detail;
+  };
+};
+
+onMount(() => {
+  // @ts-expect-error Debug function.
+  window.DEBUG_addObstacles = () => {
+    for (let i = 0; i < 100; i += 1) {
+      const x = (i % 10) / 6500;
+      const y = (Math.trunc(i / 10)) / 6500;
+      $obstacles = [
+        ...$obstacles,
+        createObstacle(`Obstacle ${$obstacles.length + 1}`, $mapCenter.lng + x, $mapCenter.lat + y),
+      ];
+    }
+  };
+});
+
 </script>
 
-<nav class='min-w-[8rem] mr-2'>
-  <h3 class='text-xs py-1.5 mb-1.5 border-b border-light'>Obstacles</h3>
-  <ul class='mb-4 font-mono'>
-    {#if $obstacles.length === 0}
-      <li class='text-xs text-subtle-2 font-sans py-2'>None</li>
-    {/if}
+<nav class='w-80'>
```

Comment:
I don't see any anchor links in this component. Is `<nav>` the best semantic choice here?